gw_eccentricity/eccDefinition.py

```diff
@@ -154,15 +155,27 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
                 https://github.com/vijayvarma392/gw_eccentricity/wiki/NR-investigation-to-set-default-number-of-orbits-to-exclude-before-merger
                 Default: 2.
 
+        precessing: bool, default=False
+            Whether the system is precessing or not. For precessing systems,
+            the `dataDict` should contain modes computed in the intertial frame
+            which are rotated internally to obtain the same in the coprecessing
+            frame. Finally the modes in the coprecessing frame are used for
+            measuring eccentricity.
```

Comment:
Does the following rephrasing work?

![image](https://github.com/vijayvarma392/gw_eccentricity/assets/44096766/1cd87454-6f95-42a9-b37b-f3e443860328)

Reply:
Hmm, I thought you were not doing the rotation in the package for now? Could you point me to where this is done? Also, "For precessing....should contain modes contained in the inertial frame" suggests you want something else for nonprecessing.

Reply:
Yes, it is not done right now. Right now `precessing=True` should not work. So technically the above documentation is not correct right now. But this is what we intend to do in future when precessing cases will be supported. Should I add a note here that only non-precessing case is supported right now?

The second para says for nonprecessing system there is no distinction between inertial and coprecessing.

Basically what I want to say is that if `precessing=True`, I want to ensure that the modes are in the intertial frame otherwise, I don't know whether I need to apply rotation or not.

Reply:
I don't think the docs should say things that aren't done yet, just say things like: For precessing, the input waveform should already be in the coprecessing frame..stuff like that. You can update it when the code actually handles inertial frame precessing waveforms.

Reply:
What do you mean `precessing=True` should not work? Why can't we just pass it a copr waveform and have it work for that?

Reply:
> What do you mean `precessing=True` should not work? Why can't we just pass it a copr waveform and have it work for that?

It should in principle work now. But not tested as explained above.